backend/.editorconfig

```
@@ -0,0 +1,7 @@
+root = true
+
+[*.{kt,kts}]
+charset=utf-8
+end_of_line=lf
+indent_style=tab
```

```suggestion
indent_size = 4
indent_style = space
```
space로 하는게 offical kotlin style인 것 같습니다.